### PR TITLE
Project part 1

### PR DESCRIPTION
**Pull request by EotronRM**

### CODE REVIEW

File: team_one_behave/features/project/pivotal_tracker/project_history_smoke.feature [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@smoke
+Feature: Project History
+
+  Scenario: Get an specified project history
+    Given I set up a "GET" request to "/projects/2242582/history/days" endpoint
+    When I send the request
+    Then I get a "200" status code as response
```
Comment (abelgustavo) [inline team_one_behave/features/project/pivotal_tracker/project_history_smoke.feature]: add final line

File: team_one_behave/features/project/pivotal_tracker/project_integration_smoke.feature [hunk 2]
```diff
@@ -0,0 +1,27 @@
+@smoke
+Feature: Project Integration
+
+  Scenario: Get an specified project integration
+    Given I set up a "GET" request to "/projects/2242582/integrations" endpoint
+    When I send the request
+    Then I get a "200" status code as response
+
+#  Scenario: Post an specified project integration
```
Comment (abelgustavo) [inline team_one_behave/features/project/pivotal_tracker/project_integration_smoke.feature]: remove comment code

File: team_one_behave/features/project/pivotal_tracker/project_history_smoke.feature (hunk 1 above)
Comment (DennisMont) [inline team_one_behave/features/project/pivotal_tracker/project_history_smoke.feature]: Add a line at the end of the feature

File: team_one_behave/schemas/pivotal/project_schema.json [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+      "$id": "#/properties/enable_following",
+      "type": "boolean",
+      "title": "The Enable_following Schema",
+      "default": false,
+      "examples": [
+        true
+      ]
+    }
+  }
+}
```
Comment (abelgustavo) [inline team_one_behave/schemas/pivotal/project_schema.json]: add final line

File: team_one_behave/features/project/pivotal_tracker/project_integration_smoke.feature [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+#    """
+#    {
+#      "active":false,
+#      "base_url":"http://some.th/ing",
+#      "name":"something",
+#      "type":"other"
+#    }
+#    """
+#    When I send the request
+#    Then I get a "200" status code as response
```
Comment (DennisMont) [inline team_one_behave/features/project/pivotal_tracker/project_integration_smoke.feature]: Add a line at the end of the feature.

**Review by abelgustavo — CHANGES_REQUESTED** [review]
please update the review

File: team_one_behave/features/steps/project_pivotal_steps.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+
+@step("I set up the header")
+def step_impl(context):
+    logger.info("Add Headers to request")
+    header = json.loads(context.text)
+    data = context.client.get_headers()
+    for key in header.keys():
+        data[key] = header[key]
+    context.client.set_headers(data)
```
Comment (DennisMont) [inline team_one_behave/features/steps/project_pivotal_steps.py:22]: Add a line at the end of the code.

**Review by abelgustavo — APPROVED** [review]

**Pull request merged**